{"name": "test_raise_covid19_exception_should_pass", "status": "passed", "start": 1650332338891, "stop": 1650332338891, "uuid": "10fbb068-0172-4e4b-b095-dda5f47d1010", "historyId": "03ab836c6162afe9e3db1f479d8534d8", "testCaseId": "0e85c644bb1577955e01fa0f933069f5", "fullName": "coronavstech.companies.tests.test_api#test_raise_covid19_exception_should_pass", "labels": [{"name": "tag", "value": "django_db"}, {"name": "parentSuite", "value": "coronavstech.companies.tests"}, {"name": "suite", "value": "test_api"}, {"name": "host", "value": "youssfi"}, {"name": "thread", "value": "17542-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "coronavstech.companies.tests.test_api"}]}